tests/conftest.py

```
@@ -0,0 +1,149 @@
+def sample_proxy_file(tmp_path):
+    """Create a sample proxy file."""
+    proxy_file = tmp_path / "proxies.txt"
+    proxy_file.write_text("""
+192.168.1.1:8080
+192.168.1.2:8080:user1:pass1
+192.168.1.3:8080:user2:pass2
+    """.strip())
+    return proxy_file
+
+@pytest.fixture
+def mock_aioresponse():
```

Ruff (0.8.2)

`mock_aioresponse` is too complex (12 > 10)

(C901)